experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormFeatureValue/RichFeatureValueEditor.tsx

```diff
@@ -11,9 +11,12 @@ import React, { useCallback, useMemo, useRef } from "react";
 import { useController, useFormContext } from "react-hook-form";
 import { FeatureValueEditorProps } from "src/components/PageEditBranches/FormBranches/FormFeatureValue/props";
 import {
+  fmlLinter,
   schemaAutocomplete,
   schemaLinter,
-} from "src/components/PageEditBranches/FormBranches/FormFeatureValue/schema";
+} from "src/components/PageEditBranches/FormBranches/FormFeatureValue/validators";
+
+const allowFmlLinting = false;
```

Comment:
please file a ticket to remove this demo stuff and add a comment here with a TODO

Reply:
These are already covered in the other Jira tickets, so I'll add a todo 👍